### PR TITLE
Issue #77: Remove areas we have decided not to use

### PR DESCRIPTION
**Pull request by nengard**

### CODE REVIEW

**Comment by nengard** [comment]
@pmkovar can you take a look, I think I made changes correctly.

File: topics/introduction.adoc [hunk 1]
```diff
@@ -12,12 +12,7 @@ Customer Content Services (CCS) is a Red Hat team responsible for producing docu
...
 
-[[ccg-evaluating-the-framework]]
-=== Evaluating the Framework
-
-Placeholder text.
-
-[[ccg-checklist]]
-==== Checklist
+[[ccg-contacts]]
+== Contacts
```
Comment (pmkovar) [inline topics/introduction.adoc]: I guess this is a placeholder for #67?

File: topics/introduction.adoc [hunk 2]
```diff
@@ -12,12 +12,7 @@ Customer Content Services (CCS) is a Red Hat team responsible for producing docu
 
 In principle, single-streamed documentation will be worked on completely in the upstream and brought down to the Customer Portal wholesale. In some projects, this process eliminates the overhead of maintaining two separate repositories for documentation content. This framework guides you through the process necessary for deciding if a community is a viable single-stream partner, setting up your repositories, choosing the right tools, and more.
 
-[[ccg-evaluating-the-framework]]
-=== Evaluating the Framework
-
-Placeholder text.
-
-[[ccg-checklist]]
-==== Checklist
```
Comment (pmkovar) [inline topics/introduction.adoc]: Wrt #63, do we want to retain Checklist here?

File: topics/introduction.adoc (hunk 1 above)
Comment (nengard) [reply topics/introduction.adoc]: Yes, I moved it from later in the guide to the Introduction.

File: topics/introduction.adoc (hunk 2 above)
Comment (nengard) [reply topics/introduction.adoc]: I think it has to be *Something* Checklist - and I also think it should probably come at the end - as if to say 'did you do all these things?'  Does that make sense?

Comment (JStickler) [reply topics/introduction.adoc]: I usually want a checklist at the <i>beginning</i>, to give me an overview of what I'm getting myself into.

Comment (nengard) [reply topics/introduction.adoc]: Oh, okay :) I'm not picky :)

**Pull request merged**